### ISSUE TITLE
prompt for administrative privileges on upgrade has blank, mis-aligned button

### ISSUE THREAD

**Issue by glyph** [docker/for-mac#2856]
Please, check https://docs.docker.com/docker-for-mac/troubleshoot/.
Issues without logs and details cannot be debugged, and will be closed.

  - [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have submitted Diagnostics
  - [x] I have included the Diagnostics ID in this Issue 1D83BF2B-37CD-4516-9408-59A7F75F39AB

### Expected behavior

A dialog box with an "OK" button, correctly aligned, that looks like something I would be comfortable typing my admin password into

### Actual behavior

the dialog below that makes me do a double-take and wonder if I'm running malware, every time

![messages image 3812622845](https://user-images.githubusercontent.com/716529/39392136-4f7858a0-4a65-11e8-8e2d-5b21dddccfe6.png)

### Information

Docker for Mac: version: 18.05.0-ce-rc1-mac63 (7e2bb492d35dc35ec0037d8024b7e7be57fc72a9)
macOS: version 10.13.4 (build: 17E202)
logs: /tmp/1D83BF2B-37CD-4516-9408-59A7F75F39AB/20180427-215214.tar.gz
[OK]     vpnkit
[OK]     virtualization hypervisor
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     files
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     disk

**Comment by akimd** [comment]
Thanks for the report.

This is a know issue, which, unfortunately, is out of our reach, as we're calling a system routine to display this.  Maybe we'll understand this some day, meanwhile we have more severe issues to address.

**Comment by glyph** [comment]
Just curious - if you are calling a system routine to display this, why is Docker the only app that exhibits this behavior?

**Comment by glyph** [comment]
(The only app I use, at any rate; a number of others prompt for administrative privilege escalations in a similar way, but the button looks correct)

**Comment by mafredri** [comment]
I just updated from `18.05.0-ce-X` to `18.06.0-ce-rc3-mac68` and was greeted by the misaligned button, however:

1. When I tried installing stable a while back, misaligned button
2. When I switched (IIRC uninstall / install edge manually) to edge (18.05.0-ce), aligned button
3. When I just now updated to 18.06.0-ce-rc3-mac68, misaligned button
    1. I clicked cancel & exit
    2. Relaunch docker, aligned button

Super weird.

I would like to add that generally I feel very untrusting towards software that has any weirdness or flakyness in the privilege prompt. Dropbox (at least at some point) implemented their own and stored your password in their own database (locally), very much not cool. As such, I feel this issue is quite important as it decreases the trust in the software and triggers suspicion, just my two cents.
